src/main/java/org/dataloader/DataLoader.java

```diff
@@ -64,8 +65,9 @@
 public class DataLoader<K, V> {
 
     private final DataLoaderHelper<K, V> helper;
-    private final CacheMap<Object, CompletableFuture<V>> futureCache;
     private final StatisticsCollector stats;
+    private final CacheMap<Object, V> futureCache;
```

Comment:
so maybe `FutureCache` is a better name than `CacheMap`?

Reply:
Yes but that would definitely break everyone using it.

I want that better name but I am willing to wear the non breaking API change